open-sphere-base/core/src/main/java/io/opensphere/core/geometry/GeometryRegistryImpl.java

```diff
@@ -34,8 +34,8 @@
     private final TLongObjectHashMap<Object> myDataModelMap = new TLongObjectHashMap<>();
 
     /**
-     * Whether to add geometries to the data model map. Nothing currently uses the data model map functionality so it's set to
-     * false to save memory.
+     * Whether to add geometries to the data model map. Nothing currently uses
+     * the data model map functionality so it's set to false to save memory.

```

Comment:
Since it's actually set to true now, should this comment change?

Reply:
Fixed by removing the field entirely.